bin/src/_main.rs

```diff
@@ -213,6 +214,7 @@ pub async fn _main(
 
             k8s_event_stream
         }
+        Err(K8sError::K8sNotInClusterError()) => None,

```

Comment:
Probably best to have a debug! message here rather than completely swallowing it, just in case someone is doing something weird with the agent

Reply:
done